dockers/screenshotter/screenshotter.js

```diff
@@ -277,6 +281,8 @@ function buildDriver() {
         ffProfile.setPreference("browser.startup.page", 0);
         const ffOptions = new firefox.Options().setProfile(ffProfile);
         builder.setFirefoxOptions(ffOptions);
+    } else if (opts.browser === "chrome") {
+        builder.setChromeOptions(new chrome.Options().setProxy(null));
```

Comment:
This seems reasonable, but did something necessitate this change?

Reply:
@kevinbarabash This was in attempt to fix Chrome screenshotter taking too long (~20 mins) occasionally. [Some posts](https://stackoverflow.com/questions/58606020/selenium-webdriver-chromedriver-slow-to-launch-why) suggested it's a proxy issue, so I tried disabling proxy, but it turns out the issue is still reproducible. Then I've looked at Docker logs and it showed `Timed out receiving message from renderer`. [This post](https://stackoverflow.com/questions/48450594/selenium-timed-out-receiving-message-from-renderer) suggested adding `--disable-gpu` argument, and the issue is not reproducible for about 15 retries.